.github/workflows/build-docker-manifest.yaml

```
@@ -0,0 +1,88 @@
+          registry: ${{ inputs.docker_gcp_registry }}
+          image: ${{ inputs.docker_image_name }}
+          version: ${{ inputs.docker_debug_version }}
+      - name: Create multi-arch docker manifest - latest
+        if: inputs.version_type == 'pr'
+        uses: hoprnet/hopr-workflows/actions/multi-arch-manifest@ausias/fix-actions
+        with:
+          registry: ${{ inputs.docker_gcp_registry }}
+          image: ${{ inputs.docker_image_name }}
+          version: latest
+      - name: Log in to Docker Hub
+        if: inputs.version_type == 'release' && secrets.docker_hub_username != '' && secrets.docker_hub_token != ''
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)